Assets/Plugins/iOS/BidMachineUnityBridge.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  BidMachineiOSBridge.swift
+//  UnityFramework
+//
+//  Created by Dzmitry on 27/09/2024.
+//
+
+import Foundation
+import BidMachine
+
+let iOSUnityBridge = BidMachineUnityBridge(instance: .shared)
+
+final class BidMachineUnityBridge {
+    let rewardedBridge: RewardedAdBridge
+
+    var isInitialized: Bool {
+        instance.isInitialized
+    }
+    
+    private let instance: BidMachineSdk
+    
+    init(instance: BidMachineSdk) {
+        self.instance = instance
+        self.rewardedBridge = RewardedAdBridge(instance: instance)
+    }
+    
+    func initialize(with sourceId: String) {
+        instance.initializeSdk(sourceId)
+    }
+
+    func setEndpoint(url: String) {
+        instance.installFeature(Constant.endpointKey, url)
+    }
+    
+    func setLoggingEnabled(_ enabled: Bool) {
+        instance.populate {
+            $0.withLoggingMode(enabled)
+        }
+    }
+    
+    func setTestEnabled(_ enabled: Bool) {
+        instance.populate {
+            $0.withTestMode(enabled)
+        }
+    }
+    
+    func setTargeting(parameters: TargetingParameters) {
+        instance.targetingInfo.populate { info in
+            info.withUserId(parameters.userId)
+            info.withUserGender(parameters.gender.asBMGender)
+            info.withUserYOB(UInt32(parameters.birthdayYear))
+            info.withKeywords(parameters.keywords.first ?? "") // FIXME: how to transform array of keywords?
```

Comment:
Should be comma joined, but I'm not sure about `","` or `", "`

Reply:
I reviewed the Android SDK implementation [here](https://github.com/bidmachine/BidMachine-Android-SDK/blob/master/bidmachine/src/main/java/io/bidmachine/TargetingParams.java#L240) and replicated the same behavior